### ISSUE TITLE
need some suggestioins about fine tune

### ISSUE THREAD

**Issue by itstao** [AlexeyAB/darknet#2635]
I have trained a network AlexNet on my  **dataset A** including 10 classes. The size of images is fixed 256 by 256. Now I have a similar **dataset B** but with a varied size(eg:49 by 89, 99 by 120 ), no bigger than the previous size 256*256. My idea is that use the pre-trained weights(**dataset A**) to keep training on the **dataset B** , and I only want to update the last layer's weights (FC layer). 
1.What should I do to proceed it ?
2.Should I change the height and width in the cfg file?
Hope somebody can give some advice. Best regards

**Comment by AlexeyAB** [comment]
You can do partial to get pre-trained weights (with 11 layers) that isn't depend on last layer:
`./darknet partial cfg/alexnet_A.cfg alexnet_A.weights alexnet.conv.11 11`

And then train your new model B by using this pre-trained weights
`darknet.exe classifier predict cfg/imagenet1k.data cfg/alexnet_B.cfg alexnet.conv.11`

**Comment by itstao** [comment]
@AlexeyAB  Hi,thank you for response.I have been trying using stopbackward=1 to freeze some layers like this:

[connected]
output=4096
activation=relu
stopbackward=1 #add  2019.3.21
[dropout]
probability=.5
#stopbackward=1 #add  2019.3.19

[connected]
output=10
activation=linear

[softmax]
groups=1

I got two questions:
1.Does this method realize the same effect? 
2.Does it really use **predict** instead of **train**?  `darknet.exe classifier predict cfg/imagenet1k.data cfg/alexnet_B.cfg alexnet.conv.11` 
thank you !
